srcpkgs/OpenRCT2/template

```diff
@@ -5,10 +5,11 @@ pkgname=OpenRCT2
 _objects_version=1.0.21
 _titles_version=0.1.2c
 version=0.3.3
-revision=1
+revision=2
 build_style=cmake
 configure_args="
  -DOPENRCT2_VERSION_TAG=${version}
+ -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

Comment:
Would you please check with `CXXFLAGS="-DNDEBUG"` instead?

Reply:
How is that preferable to setting it via CMake build type?

Reply:
Some people want to build their own package with `-O1` or `-O3` and/or disable `-g`, I suppose.
Anyway, I don't know which kitchen sink would be added into `RelWithDebInfo` in the future by CMake.

Reply:
Ah, I researched more and I understand the problem now. Unfortunate that CMake does not deal with compiler flag precedence well.

Reply:
Okay, I confirmed `-NDEBUG` works properly.